plugins/repositories/core/pom.xml

```diff
@@ -109,18 +108,6 @@
       <version>${javax.servlet-api.version}</version>
       <scope>provided</scope>
     </dependency>
-    <dependency>
```

Comment:
This is *not* in a dependency management. Double check removing the dependencies.